test/acceptance/setup-terraform/main.tf

```diff
@@ -5,6 +5,10 @@ provider "aws" {
   region = var.region
 }
 
+provider "hcp" {
+  project_id = "6a513c39-907d-4a92-9159-3b03bd00ecda"
```

Comment:
:thinking: I can't run the acceptance tests. I get an error during `terraform apply` that looks related to this.

```shell
setup-terraform$ terraform apply ...
...
╷
│ Error: unable to fetch project "6a513c39-907d-4a92-9159-3b03bd00ecda": [GET /resource-manager/2019-12-10/projects/{id}][403] ProjectService_Get default  &{Code:7 Details:[] Message:}
│ 
│   with provider["registry.terraform.io/hashicorp/hcp"],
│   on main.tf line 8, in provider "hcp":
│    8: provider "hcp" {
│ 

```

Reply:
Projects seem to be a recent-ish addition to HCP. Or at least having multiple projects is.

The Terraform provider indicates that you may need to set project_id in some cases:

> Note:
If a [Project-Level Service Principal](https://cloud.hashicorp.com/docs/hcp/admin/iam/service-principals) is used, specify the default project_id in your provider configuration.

We probably need to add an input variable to setup-terraform so that we can specify an HCP project id if necessary. What do you think?